src/pages/DataFetching/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import BackToHome from "../../components/BackToHome";
+import PageNav from "../../components/PageNav";
+import FetchAPI from "../../components/DataFetching/FetchAPI";
+import Axios from "../../components/DataFetching/Axios";
+import Search from "../../components/DataFetching/Search";
+
+import "./dataFetching.scss"
+
+const DataFetching = ({ page }) => {
+  const navData = [
+    {
+      path: "/datafetching/fetch-api",
+      title: "Fetch API"
+    },
+    {
+      path: "/datafetching/axios",
+      title: "Axios"
+    },
+    {
+      path: "/datafetching/search",
+      title: "Search"
+    },
+  ];
```

Comment:
The `navData` can be put outside the function component, so we won't be redeclaring it whenever the component re-renders.

Reply:
updated, moved navData outside of function component